scripts/qa_crowbarsetup.sh

```diff
@@ -3273,12 +3273,16 @@ function oncontroller_manila_generic_driver_setup()
     fi
 
     if [ "`openstack server show --f value -c status manila-service`" != ACTIVE ]; then
-        fixed_net_id=`neutron net-show fixed -f value -c id`
-        timeout 10m nova boot --poll --flavor 100 --image manila-service-image \
-            --security-groups $sec_group,default \
-            --nic net-id=$fixed_net_id manila-service
-
-        [ $? != 0 ] && complain 43 "nova boot for manila failed"
+        # Removed boot --poll test until a proper fix is in place.
+        # https://trello.com/c/m9a3MQYS/173-c6-nova-boot-for-manila-failed
+        if test ${hacloud} -ne 1; then
```

Comment:
I think you are right. Just checked one of the nodes where we usually run mkcloud and sh seems to be mapped to bash. I won't change it for now.
```
mkcloud2:~ # /bin/sh --version
GNU bash, version 3.2.57(1)-release (x86_64-suse-linux-gnu)
Copyright (C) 2007 Free Software Foundation, Inc.
```